tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -298,6 +298,24 @@ private void OnEndRequest(object sender, EventArgs eventArgs)
                             }
 
                             CookieAnalyzer.AnalyzeCookies(app.Context.Response.Cookies, IntegrationId);
+                            if (HttpRuntime.UsingIntegratedPipeline &&
+                                _canReadHttpResponseHeaders)
+                            {
+                                try
+                                {
+                                    HeaderAnalyzer.AnalyzeHeaders(app.Context.Response.Headers, IntegrationId.AspNetCore);
```

Comment:
We already have a ReturnedHeadersAnalyzer class. I guess that it would make sense to analyze the returned headers in that class. We could extend the method ReturnedHeadersAnalyzer.Analyze() called above to include that functionality as well.

Reply:
Yes, you are right. This `ReturnedHeadersAnalyzer` class was not merged when I started this implementation. And It makes sense to unify all header vuln analyzers in the same place. Although I'd rather call it `ResponseHeadersAnalyzer`